pkg/server/server.go

```diff
@@ -19,13 +19,12 @@ package server
 import (
 	"crypto/tls"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"log"
 	"net/http"
 	"regexp"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/ec2provider"
```

Comment:
Can you move this to third party imports?

Reply:
ack